cmd/deploy/proxy.go

```diff
@@ -340,7 +340,8 @@ func (ph *proxyHandler) translateMetadata(body map[string]json.RawMessage) error
 func (ph *proxyHandler) translateDeploymentSpec(body map[string]json.RawMessage) error {
 	var spec appsv1.DeploymentSpec
 	if err := json.Unmarshal(body["spec"], &spec); err != nil {
-		return fmt.Errorf("could not process deployment spec: %s", err)
+		oktetoLog.Info(err)
```

Comment:
I'd add a descriptive wrapper
```suggestion
		oktetoLog.Infof("error unmarshalling deployment spec on proxy: %w")
```

Reply:
No, it will be stored on the doctor file and only will be shown to the user if the user specifies `loglevel=info`

Reply:
yes, I think so @ifbyol. I think it's enough to add the detail of the error message with `%s + err.Error()`